### DISCUSSION TITLE
I have several small HDD scattered around; I want to make them one big "virtual HDD" to store all my stuff; I care about size most and data integrity; does juicefs suits my need?

### DISCUSSION THREAD

**Discussion by formula-spectre** [juicedata/juicefs#4817] [Q&A]
I can buy rpi02W with an ETH+USB HAT for each HDD, or plug them in all in the various servers;  
atm I have 3/4 160GB hdd, one 320GB, and two 500GB; I want to have one big storage box that the size is the sum of those (so 1800-ish GB), with somewhat data integrity check, a la zfs; does JuiceFS fits my needs? or should I keep looking?

**Comment by sinofool** [comment]
Have you checked btrfs?

**Reply by formula-spectre** [reply]
yes I did; I could make giant raid, but if I want to make it distributed I have to do nfs shenanigas

**Comment by davies** [comment]
JuiceFS + MinIO + Redis could be a solution for your case
